### PR TITLE
fix(migration): handle lost value last_boot, remote_addr, last_inventory_update

### PR DESCRIPTION
**Pull request by stonebuzz**
Some data are lost during migration

- ```last_boot```
- ```last_inventory_update```
- ```remote_addr```

This PR prevent this

### CODE REVIEW

File: install/update.php [hunk 1]
```diff
@@ -3099,6 +3099,69 @@ function do_biosascomponentmigration()
 {
     global $DB;
 
+    //Agent data migration -> remote_addr
+    if (
+        $DB->tableExists('glpi_plugin_glpiinventory_inventorycomputercomputers') &&
+        $DB->fieldExists('glpi_plugin_glpiinventory_inventorycomputercomputers', 'remote_addr')
+    ) {
+        //retrieve exiting
+        $query = "SELECT computers_id, remote_addr
```
Comment (trasher) [inline install/update.php:3108]: Please avoid raw SQL queries

**Review by trasher — CHANGES_REQUESTED** [review]
See comment

File: install/update.php (hunk 1 above)
Comment (stonebuzz) [reply install/update.php:3108]: done !

**Comment by stonebuzz** [comment]
> See comment

done

File: install/update.php [hunk 2]
```diff
@@ -3099,6 +3099,62 @@ function do_biosascomponentmigration()
...
+                    $input_computer['last_inventory_update'] = $data['last_inventory_update'];
+                }
+
+                if (count($input_computer) > 1) {
+                    $computer->update($input_computer);
+                }
+
+                //handle remote_addr
+                $agent = $computer->getInventoryAgent();
+                if (!is_null($agent) && !empty($data['remote_addr'])) {
```
Comment (trasher) [inline install/update.php:3144]:
```suggestion
                if (!is_null($agent) && !empty($data['remote_addr']) && empty($agent->fields['remote_addr'])) {
```

File: install/update.php [hunk 3]
```diff
@@ -3099,6 +3099,62 @@ function do_biosascomponentmigration()
...
+                'last_boot',
+                'last_inventory_update'
+            ],
+            'FROM' => 'glpi_plugin_glpiinventory_inventorycomputercomputers'
+        ]);
+
+        foreach ($computers_iterator as $data) {
+            $computer = new Computer();
+            if ($computer->getFromDB($data['computers_id'])) {
+
```
Comment (trasher) [inline install/update.php:3124]:
```suggestion
```

**Review by cedric-anne — COMMENTED** [review]
There is already something that is handling `last_inventory_update` in `update.native.php`.

For instance:
```php
        $DB->queryOrDie(
            "UPDATE `glpi_computers` AS `computers`
            INNER JOIN (
              SELECT
                `computers_id`,
                `last_inventory_update`
              FROM `glpi_plugin_glpiinventory_inventorycomputercomputers`
          ) AS `plugin_computers` ON `plugin_computers`.`computers_id` = `computers`.`id`
          SET
              `computers`.`last_inventory_update` = `plugin_computers`.`last_inventory_update`
          ;"
        );
```

**Pull request closed without merging**